Test/catch

kDAJIN opened the pull request:
## 연관 이슈
- closes #136 

## 해당 작업을 한 동기/이유는 무엇인가요?
- 캐치 도메인 테스트 코드 작성 

## 리뷰어에게 작업 또는 변경 내용을 상세히 설명해주세요
- [ ] 캐치 등록 테스트
    - 캐치가 정상적으로 등록될 경우
    - 뒹글의 오너가 아닌 사람이 캐치를 등록하려 할 경우 예외 발생
- [ ] 캐치 삭제 테스트
    - 정상적으로 캐치가 삭제될 경우 딜리티드엣 업데이트
    - 이미 이전에 캐치를 단 이력이 있는 경우 예외 발생
    - 뒹글의 오너가 아닌 사람이 삭제를 시도할 경우 예외 발생

kDAJIN committed:
test:캐치 도메인 테스트 코드 작성중

adorecamus committed:
Fix: CatchServiceDBTest 데이터 삭제 오류

- 각 테스트 후 데이터 삭제 순서 변경 (참조되는 entity가 후순위로 오도록)

rugii913 reviewed (approved):
고생 많으셨습니당
conflict 부분 해결하시면 될 거 같습니당

kDAJIN committed:
Merge branch 'dev' into test/catch

rugii913 committed:
test: userId 대신 userLink를 사용하면서 테스트 깨진 부분 수정

- userId 대신 userLink를 사용하면서 테스트 깨진 부분 수정
- 현재 실패하는 테스트는 "커서값이 전달 되었을 경우 커서 이전 뒹글 부터 개인 페이지 조회"

kDAJIN committed:
Merge branch 'dev' of https://github.com/dooingledooingle/dooingle into test/catch

kDAJIN committed:
캐치오류 수정중

kDAJIN committed:
errorfix:커서값 전달 시 개인페이지 조회 테스트 코드 수정

adorecamus committed:
Fix: Emitter 연결 후 데이터 받을 수 있도록 대기
